apps/api/src/db/schemas/applications.ts

```diff
@@ -23,7 +23,7 @@ import { applicationContents } from './applicationContents.ts';
 // TODO: Integrate w/ TS
 // import { ApplicationStates } from 'pcgl-daco/packages/data-model';
 
-const statesEnum = pgEnum('applicationStates', [
+export const statesEnum = pgEnum('application_states', [
```

Comment:
gotta keep consistent with agreements etc
so we have to either add application to names in this file, or remove agreements there (which looks less descriptive)

better being extra verbose overall, than not enough
```suggestion
export const applicationStatesEnum = pgEnum('application_states', [
```